ivy/functional/frontends/numpy/mathematical_functions/rounding.py

```diff
@@ -33,6 +33,28 @@ def _ceil(
     return ret
 
 
+@handle_numpy_out
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@handle_numpy_casting
+@from_zero_dim_arrays_to_scalar
+def _floor(
+    x,
```

Comment:
This is not a private function in numpy. Kindly use `floor` as is.

Reply:
Done.